src/pages/cli/graphql/client-code-generation.mdx

```diff
@@ -21,6 +21,8 @@ You’ll see questions as before, but now it will also automatically ask you if
 
 When you deploy your GraphQL API to the cloud, you are prompted to configure codegen. When a project is configured to generate code with codegen, it stores all the configuration `.graphqlconfig.yml` file in the root folder of your project.
 
+Models are not automatically generated by codegen on an `amplify push`.
```

Comment:
Why aren't we triggering modelgen? If it's necessary for their app and they've configured it upon prompt then we should do it automatically.

Reply:
It doesn't seem like anyone has clear information on this after investigating, so I'm removing this and there will be a follow up discussion/change